Bottom: 384284f27e8fbe1feae9df31f75e2968ae412427
Top:    ce89b7c45ec87f119bf08ae9c7cdcc16d0d1aede
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-08 19:48:31 +1300

Refresh of extensible-run

---

diff --git a/testtools/tests/test_testsuite.py b/testtools/tests/test_testsuite.py
index 90ed638..5290912 100644
--- a/testtools/tests/test_testsuite.py
+++ b/testtools/tests/test_testsuite.py
@@ -179,9 +179,9 @@ class TestConcurrentStreamTestSuiteRun(TestCase):
         # Check the traceback loosely.
         self.assertThat(events[1][6].decode('utf8'), DocTestMatches("""\
 Traceback (most recent call last):
-  File "...testtools/testsuite.py", line 188, in _run_test
+  File "...testtools/testsuite.py", line 190, in _run_test
     test.run(process_result)
-TypeError: run() takes ...1 ...argument...2...given...
+TypeError: run() takes...1...argument...2...given...
 """, doctest.ELLIPSIS))
         events = [event[0:10] + (None,) for event in events]
         events[1] = events[1][:6] + (None,) + events[1][7:]
